doc/releasenotes/14_0_0_summary.md

```diff
@@ -1,5 +1,16 @@
 ## Major Changes
 
+### New query support
```

Comment:
What about Gen4 being the default? Where have we documented that?

Reply:
nice catch. I'll add it